static/css/home.scss

```diff
@@ -0,0 +1,507 @@
+}
+
+.download-firefox .button-title {
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 1em;
+  margin-bottom: 0.25em;
+}
+
+.download-firefox .button-description {
+  font-size: 18px;
+  font-weight: 300;
+  line-height: 1em;
+}
+
+/* Instruction Sections */
+
+section {
+  align-items: center;
+  display: flex;
+  min-height: 600px;
+}
+
+section:nth-child(even) {
+  background-color: #f7fdfe;
+}
+
+section .container {
+  max-width: 1100px;
+  padding: 40px 63px;
+}
+
+.section-content h3 {
+  font-size: 2.5em;
+  font-weight: 400;
+  margin: 0;
+}
+
+.section-content .bold {
+  font-weight: 500;
+}
+
+.section-content {
+  color: #4a4a4a;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  line-height: 1.61;
+  width: 36%;
+}
+
+.section-image {
+  background-size: contain;
+  display: flex;
+  height: auto;
+  justify-content: flex-end;
+  width: 50%;
+}
+
+#section-1 .section-image {
+  background-image:   url("../img/landing-screenshots_image_01.svg");
+  background-position: 100% center;
+  background-repeat: no-repeat;
+  height: 476px;
+}
+
+#section-2 .section-image {
+  background-image:   url("../img/landing-screenshots_image_02.svg");
+  background-position: 0% center;
+  background-repeat: no-repeat;
+  height: 500px;
+}
+
+#section-3 .section-image {
+  background-image:   url("../img/landing-screenshots_image_03.svg");
+  background-position: 100% center;
+  background-repeat: no-repeat;
+  height: 519px;
+}
+
+#section-4 .section-image {
+  background-image:   url("../img/landing-screenshots_image_04.svg");
+  background-position: 100% center;
+  background-repeat: no-repeat;
+  height: 533px;
+}
+
+#section-1 .section-content,
+#section-2 .section-image,
+#section-4 .section-content {
+  order: 1;
+}
+
+#section-1 .section-image,
+#section-2 .section-content,
+#section-4 .section-image {
+  order: 2;
+}
+
+
+/* Coming Soon Section */
+
+#coming {
+  min-height: 350px;
+}
+
+#coming .container {
+  max-width: 840px;
+  padding: 40px 100px;
+}
+
+.coming-content {
+  color: #4a4a4a;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  width: 45%;
+}
+
+.coming-content h3 {
+  font-size: 2em;
+  font-weight: 100;
+  margin: 0;
+}
+
+.coming-content p {
+  font-size: 0.9em;
+  line-height: 1.61;
+  margin-bottom: 0;
+}
+
+.coming-icons {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+}
+
+.coming-icons span {
+  color: #4a4a4a;
+  font-size: 2em;
+  margin: 0 33px;
+  position: relative;
+  top: 16px;
+}
+
+.coming-icons div {
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 80px;
+  width: 80px;
+}
+
+
+
+/* Responsive */
+
+@media (max-width: 1120px) and (min-width: 769px) {
+
+  .banner {
+    min-height: 500px;
+  }
+
+  .banner-image {
+    top: 0;
+    background-size: 90% auto;
+    left: 0;
+    bottom: 0;
+    right: -10%;
+  }
+
+  section {
+    min-height: 500px;
+  }
+}
+
+@media (max-width: 768px) {
```

Comment:
Not sure if you want to use the `@include respond-to("small") {` style helpers for consistency.
🤷‍♀️